azure/cert_test.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
```

Comment:
Wrong year

Reply:
Good catch. Will fix. Was a bit in a rush and didn't run the checks. Thank You!

Reply:
trailing spaces removed, code pylintified, year updated.

Reply:
Nice!
Like I said, automatic check cannot complain about commit subject starting with "Demo" as it could be interpreted as imperative mood verb, and the check is inconclusive. Anyway, the "Add demo script..." -subject is fine.
Please make it a one commit. I.e. squash intelligently in your local branch, and force push to the PR branch.
Do not use squash and merge feature in github.